lib/core/CDataFrameRowSlice.cc

```diff
@@ -138,7 +138,7 @@ operator=(const CDataFrameRowSliceHandle& other) {
 }
 
 CDataFrameRowSliceHandle& CDataFrameRowSliceHandle::
-operator=(CDataFrameRowSliceHandle&& other) = default;
+operator=(CDataFrameRowSliceHandle&& other) noexcept = default;
```

Comment:
Even though an `= default` move assignment would usually be `noexcept` automatically if possible, this one needs the explicit declaration as `= default` is in the implementation file so when the compiler only sees the contents of the header it wouldn't know whether `noexcept` was possible.

Reply:
👍 I'd missed this one with the earlier change to make move constructors and assignment `noexcept`.